packages/server/src/app.ts

```diff
@@ -4,7 +4,7 @@ import logger from 'koa-logger';
 import json from 'koa-json';
 import serve from 'koa-static';
 
-import { MyType, add } from 'shared';
+import { MyType, add } from 'shared'; // eslint-disable-line
```

Comment:
why disable linting this line?

Reply:
I was getting `import/no-extraneous-dependencies` lint error for this line complaining that `shared` isn't list in package.json. It seems eslint doesn't work well with yarn workspace without knowing that it's importing the internal shared package. I just disabled it for now since i wasn't aware of any workaround at this moment.